pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/common/Felles.kt

```diff
@@ -110,6 +111,16 @@ object Felles {
             )
     }
 
+    data class KronerTextBold(val kroner: Expression<Kroner>) : TextOnlyPhrase<LangBokmalNynorskEnglish>() {
```

Comment:
Trenger vi en egen frase for dette? Kan ikke vi ha et felt på `KronerText` for font med default verdi normal?

Reply:
Hadde ikke tenkt på at det går an. Fixed.